### DISCUSSION TITLE
How to disable renderercodeintegrity on gauge/taiko/chromium

### DISCUSSION THREAD

**Discussion by sankarasai18-gauge** [getgauge/taiko#1984] [Development]
Hi, I would like to know how to disable renderercodeintegrity on gauge+taiko framework for chromium browser in order to fix "Aw Snap!" Crash in Google Chrome 78 ?. Please let me know. TIA.

**Comment by zabil** [comment]
> disable renderercodeintegrity

I am not sure what this is but have you tried finding the option to disable this while launching the browser? 

There is more info at https://docs.taiko.dev/api/openbrowser/ which links to the list of chrome launch options at https://peter.sh/experiments/chromium-command-line-switches/

**Reply by sankarasai18-gauge** [reply]
Thanks for your response. I am trying to launch the browser through my automation script and when I try manually I don't get the issue "Aw Snap!" Crash in Google Chrome 78.

![image](https://user-images.githubusercontent.com/82411044/114521234-ddaf2000-9c5f-11eb-98c5-5876236be6ba.png)

**Reply by zabil** [reply]
Since you mentioned chrome 78 I'd like to point out that Taiko only work on the latest versions(right now 88) of [chrome](https://github.com/getgauge/taiko/blob/371fac646fdae3d7b0d3e9b93b836483e2a4ce1c/package.json#L46). 

Taiko might not work on earlier versions of Chrome and it depends on specific api's of [CDP](https://chromedevtools.github.io/devtools-protocol/) which are incompatible between versions.

If you are using the latest version of Taiko and chrome can you share a script to replicate this issue?

**Reply by sankarasai18-gauge** [reply]
Thanks again. I got some pointers and fixed the issue finally by adding the below code ('--disable-RendererCodeIntegrity')

await openBrowser({ headless: false, ignoreCertificateErrors: true, args: ['--no-sandbox', '--start-fullscreen', '--disable-web-security','--disable-RendererCodeIntegrity'] })
